js/main.js

```diff
@@ -1,2 +1,27 @@
 /* global data */
 /* exported data */
+
+var $photo = document.querySelector('.photoUrl');
+var $image = document.querySelector('.img');
+
+$photo.addEventListener('input', function (event) {
+  $image.setAttribute('src', event.target.value);
+});
+
+var $form = document.querySelector('.form');
+
+$form.addEventListener('submit', function (event) {
+  event.preventDefault();
+  var entry = {
+    entryId: data.nextEntryId,
+    title: $form.elements.Title.value,
+    photoUrl: $form.elements.photoUrl.value,
+    notes: $form.elements.Notes.value
+  };
+  data.nextEntryId++;
+
+  data.entries.unshift(entry);
+  localStorage.setItem('data-profile', JSON.stringify(data));
```

Comment:
all of the local storage functionality should be done in the `data.js` file. 
This includes loading local storage when the app starts, and saving it when the app refreshes.